### DISCUSSION TITLE
Astro 5.0 发布！ | liruifengv

### DISCUSSION THREAD

**Discussion by giscus** [liruifengv/liruifengv.com#31] [Announcements]
# Astro 5.0 发布！ | liruifengv

Astro 5.0 带来了令人兴奋的新功能，例如 Astro Content Layer，可以从任何来源无缝加载内容，以及 Server Islands，可以轻松结合静态和动态个性化内容。

https://liruifengv.com/posts/astro-5/

**Comment by hongtaoh** [comment]
报告一个小错误。Astro 标签显示有 4 篇文章，但是点进来看到不止 4 篇。

另外想问一下博主，目前你的主题如果想要支持 i18n，需要额外设置很多吗？
